src/main/java/edu/hm/hafner/analysis/parser/MavenConsoleParser.java

```diff
@@ -85,7 +85,7 @@ protected boolean isLineInteresting(final String line) {
 
     private boolean isValidGoal() {
         return !(goal.contains("maven-compiler-plugin")
-                || goal.contains("maven-javadoc-plugin")); // will be captured by another parser already
+                || goal.contains("maven-javadoc-plugin") || goal.contains("compiler")); // will be captured by another parser already
     }
```

Comment:
Okay will try that !

Reply:
Will this code work to test whether each line is parsed by only one parser ?

public class MyMavenRunner {

    public static void main(final String[] args) throws IOException {
        // Defining the parsers to use
        MavenConsoleParser mavenConsoleParser = new MavenConsoleParser();
        JavacParser javacParser = new JavacParser();
        JavaDocParser javaDocParser = new JavaDocParser();

        // Runs the Maven build and capture its logs
        String mavenLogs = runMavenBuild();

        // Splits the logs into individual lines
        String[] logLines = mavenLogs.split("\n");

        // Iterates over each line in the logs and check which parser is able to extract information from it
        for (String logLine : logLines) {
            boolean parsedByMavenConsoleParser = mavenConsoleParser.isLineInteresting(logLine);
            boolean parsedByJavacParser = javacParser.isLineInteresting(logLine);
            boolean parsedByJavaDocParser = javaDocParser.isLineInteresting(logLine);

            // To Count the number of parsers that were able to parse the line
            int num = (parsedByMavenConsoleParser ? 1 : 0) + (parsedByJavacParser ? 1 : 0) + (parsedByJavaDocParser ? 1 : 0);

            // Checks that only one parser was able to parse the line
            if (num== 0) {
                System.err.println("Warning: line \"" + logLine + "\" was not parsed by any parser");
            } else if (num > 1) {
                System.err.println("Warning: line \"" + logLine + "\" was parsed by multiple parsers");
            } 
	    else{
		    System.out.println("The logLine is parsed by only one parser");} 
        }
    }

Reply:
I am really sorry since I am stuck with few of my assignment deadlines from my university, I am not able to focus into this .Will look into the issue again on Friday.